common/defs/support.json

```diff
@@ -204,7 +202,12 @@
       "misc:XRP": "2.6.1",
       "misc:XTZ": "2.6.1",
       "misc:tADA": "2.6.1",
-      "misc:tXRP": "2.6.1"
+      "misc:tXRP": "2.6.1",
+      "nem:BREEZE": "2.6.1",
+      "nem:DIM": "2.6.1",
+      "nem:DIMTOK": "2.6.1",
+      "nem:PAC:CHS": "2.6.1",
+      "nem:PAC:HRT": "2.6.1"
```

Comment:
@tomasklim did you add these? I don't recall touching this.

Reply:
I dont think so, but yes remove it and add it back to unsupported , same for T3B1

Reply:
done